src/ray/gcs/gcs_server/gcs_autoscaler_state_manager.cc

```diff
@@ -224,37 +271,37 @@ void GcsAutoscalerStateManager::GetNodeStates(
         gcs_node_info.state_snapshot().node_activity());
 
     // The node is alive. We need to check if the node is idle.
-    auto const &node_resource_data = cluster_resource_manager_.GetNodeResources(
-        scheduling::NodeID(node_state_proto->node_id()));
-    if (node_resource_data.is_draining) {
+    auto const &node_resource_item =
```

Comment:
is it possible to have a node in GcsNodeManager, but not yet having resource data reported here? so `find` returns `end`?